index.html

```diff
@@ -100,6 +100,46 @@ <h2>Welcome to MyPetSpotlight!</h2>
       </div>
     </div>
 
+    <div class="card">
+      <img src="./assets/images/pusho.jpeg" alt="description of your picture">

```

Comment:
Everything looks good apart from `alt="description of your picture"` which should be an actual description of your picture like "picture of the cat Pusho"